terraform/projects/app-ecs-services/task-definitions/alertmanager-server.json

```diff
@@ -9,12 +9,13 @@
       {
         "containerPort": 9093,
         "hostPort": 9093
+      },
+      {
+        "containerPort": 9094,
+        "hostPort": 9094
       }
     ],
-    "command": [
```

Comment:
Could we keep this how it was before so that `config.file` and `web.external-url` are still defined in the task definition (it feels nice to keep it in there rather than move it into Terraform which is less related and seems to have all the different commands spread over various parts of the code)? We could then have something like:

```
"command": [
     "--config.file=/etc/alertmanager/alertmanager.yml",
     "--web.external-url=${alertmanager_url}"
     "${peer_commands}"
]
```

Reply:
This is generally not a bad idea, I have not considered the concept myself. The idea behind putting the args in terraform is having a single point of reference. This simplifies maintenance a tiny little bit in that you have one point of reference.

I will test this. i suppose my only question is how does 'alertmanager_url' get passed to the config file ?